test/specs/rules/disallow-comma-before-line-break.js

```diff
@@ -60,4 +60,46 @@ describe('rules/disallow-comma-before-line-break', function() {
     it('should not report comma placement in single line object declaration', function() {
         assert(checker.checkString('var a = {a:1, c:3};').isEmpty());
     });
+
+    describe('mode option', function() {
+        describe('ignoreFunction value', function() {
+            beforeEach(function() {
+                checker.configure({
+                    disallowCommaBeforeLineBreak: {
+                        mode: 'ignoreFunction'
```

Comment:
Example of rule using allExcept: https://github.com/jscs-dev/node-jscs/blob/f55bbae8bfbd8d57a4f756193d824e6f398ff7a7/lib/rules/require-space-after-line-comment.js#L10